event/src/main/java/com/bazaarvoice/emodb/event/db/astyanax/SlabAllocator.java

```diff
@@ -1,6 +1,11 @@
 package com.bazaarvoice.emodb.event.db.astyanax;
 
+import com.google.common.collect.PeekingIterator;
+
+import java.nio.ByteBuffer;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.nio.ByteBuffer'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AUselessImportCheck)